### PR TITLE
Bug 1129801 - discovery Bluetooth devices automatically while goes into Bluetooth panel, r=arthur

### PR DESCRIPTION
**Pull request by ian-liu**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[ian-liu Ian Liu (ian-liu)](https://github.com/ian-liu) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=b0be19bb303c)

**Comment by try-server-hook** [comment]
[ian-liu Ian Liu (ian-liu)](https://github.com/ian-liu) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=f85833c4b2b6)

File: apps/settings/js/panels/bluetooth/panel.js [hunk 1]
```diff
@@ -137,6 +137,18 @@ define(function(require) {
         this._updateSearchingItem(BtContext.discovering);
       },
 
+      onShow: function() {
+        Debug('onShow():');
+
+        if (!this._leftApp) {
```
Comment (crh0716) [inline apps/settings/js/panels/bluetooth/panel.js]: nit: Let's set `this._leftApp` to false in `onInit` explicitly.

**Comment by try-server-hook** [comment]
[ian-liu Ian Liu (ian-liu)](https://github.com/ian-liu) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=6e3e7fa4c175)

**Comment by ian-liu** [comment]
Update: set `this._leftApp = false;` in `onInit`.

**Comment by ian-liu** [comment]
All tests are passed on try-server. We can merge the pull request now.

**Pull request merged**